airflow-core/docs/howto/define-extra-link.rst

```diff
@@ -62,6 +62,9 @@ The following code shows how to add extra links to an operator via Plugins:
 
 .. note:: Operator Extra Links should be registered via Airflow Plugins or custom Airflow Provider to work.
```

Comment:
Is this line still true though?

Reply:
Yeah i think so. I was testing so far with custom operators and/or plugins

Reply:
Shouldn't it work from any place? The initial reason we had restricted it from Airflow plugins was because it was loading user code in the webserver during the deserialization process. But now that we store those in xcoms and UI reads it from there, what's restricts this to Plugins?

Reply:
It stays "Custom Airflow Provider" though. And if the plugin thing isn't required -- then the note is just wrong :)

Reply:
Right, let me update it

Reply:
I thought of "should" => "can" but i guess it can be removed entirely